src/services/UsersService.ts

```diff
@@ -193,14 +193,17 @@ export class UsersService {
         'Content-Type': 'application/json',
         Authorization: `Bearer ${jwt}`,
       },
-      data: {
+    };
+
+    await this.axios.put(
```

Comment:
Is it the same as sending the body as the second parameter rather than sending it as the data field of the configuration?

Reply:
I would have guessed it's the same, but for some odd reason when setting the body as data in the configurations the headers where lost. Resulting in 401 responses since the jwt was not received.